src/game_config.cpp

```diff
@@ -260,6 +275,129 @@ Filesystem_Stream::OutputStream Game_Config::GetGlobalConfigFileOutput() {
 	return Filesystem_Stream::OutputStream();
 }
 
+Filesystem_Stream::OutputStream& Game_Config::GetLogFileOutput() {
+	if (!Player::player_config.log_enabled.Get()) {
+		static Filesystem_Stream::OutputStream noop_stream;
+		return noop_stream;
```

Comment:
Can we have one `noop_stream` instead of repeating this?

Reply:
wanted to avoid a static variable when it is not used but you are right that this micro optimisation makes no sense. :sweat_smile: